web/src/invite.ts

```diff
@@ -415,6 +438,66 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
             $("#generate_multiuse_invite_radio").prop("checked", true);
             $("#generate_multiuse_invite_radio").trigger("change");
         }
+
+        const invite_tips_data = generate_invite_tips_data();
+
+        const context = {
+            banner_type: compose_banner.WARNING,
+            classname: "setup_tips_warning",
+            banner_html: "",
+        };
+
+        if (!invite_tips_data.realm_has_description) {
+            context.banner_html = $t_html(
+                {
+                    defaultMessage:
+                        "You may want to <z-link>configure</z-link> your organization's login page prior to inviting users.",
+                },
+                {
+                    "z-link": (content_html) =>
+                        `<a href="#organization/organization-profile">${content_html.join("")}</a>`,
+                },
+            );
+            $("#invite-user-form .setup-tips-container .setup-tips-org-desc").html(
+                render_compose_banner(context),
+            );
+        } else if (!invite_tips_data.realm_has_user_set_icon) {
+            context.banner_html = $t_html(
+                {
+                    defaultMessage:
+                        "You may want to <z-link>upload a profile picture</z-link> for your organization prior to inviting users.",
+                },
+                {
+                    "z-link": (content_html) =>
+                        `<a href="#organization/organization-profile">${content_html.join("")}</a>`,
+                },
+            );
+            $("#invite-user-form .setup-tips-container .setup-tips-org-profile-pic").html(
+                render_compose_banner(context),
+            );
+        }
+
+        if (!invite_tips_data.realm_has_custom_profile_fields) {
+            context.banner_html = $t_html(
+                {
+                    defaultMessage:
+                        "You may want to configure <z-link-1>default new user settings</z-link-1> and <z-link-2>custom profile fields</z-link-2> prior to inviting users.",
+                },
+                {
+                    "z-link-1": (content_html) =>
+                        `<a href="#organization/organization-level-user-defaults">${content_html.join(
+                            "",
+                        )}</a>`,
+                    "z-link-2": (content_html) =>
+                        `<a href="#organization/profile-field-settings">${content_html.join(
+                            "",
+                        )}</a>`,
+                },
+            );
+            $("#invite-user-form .setup-tips-container .setup-tips-custom-profiles").html(
```

Comment:
Did you consider using the approach we take in `web/templates/compose_banner/automatic_new_visibility_policy_banner.hbs`? That's our intended approach for HTML banners, because having Handlebars do all the rendering via the `partial-block` syntax protects us from potential XSS security bugs that one risks with passing around HTML using the `{{{` syntax.

I don't think this specific PR would actually be a security bug, but we try hard as a project to avoid adding constructions that require paying attention to avoid XSS bugs.

Reply:
I had tried that approach but couldn't get it to work due to some issues with not being able to refer to the `compose_banner` partial present in the `templates/compose_banner` from the parent directory (`templates`). I'll try again to see if I can get it working or start a discussion at CZO for the same. Thanks!

Reply:
OK, yeah let's discuss in #frontend.